bundles/EcommerceFrameworkBundle/Model/AbstractFilterDefinitionType.php

```diff
@@ -64,4 +66,20 @@ public function getRequiredFilterField()
     {
         return '';
     }
+
+    /**
+     * @return Concrete[]
```

Comment:
this is semantically wrong, because these two methods are only needed for category filter types. 
so either we introduce an additional abstract class for these (which will be quite a big change that we cannot introduce for Pimcore X anymore), or we add a method exists to (Multi)SelectCategory classes

Reply:
Changed it to method_exists: https://github.com/pimcore/pimcore/pull/9067/commits/054e8f6cbd1b3dca2c1103c0e7e6a13f3bd0a53b